HelpAssistHer/help-assist-her
# Remove primary contact fields for now

## Description
In order to keep things simple for the volunteers who are verifying data, we are not going to have them ask for a primary contact right now. Therefore, I commented out the primary contact fields on the form. I decided to comment them out, rather than remove them, because eventually we will want to add it back. We also don't need to remove any code on the server side, as everything still works, the primary contact information is just ignored.

## Test Plan
1. Checkout the `master` branch
1. Click Get One Resource button to load a resource in the form
1. Fill out the primary contact fields and click Submit
1. Copy the `_id` of the resource that you just updated (you can find it in the network tab of dev tools)
1. Look up the resource in the database via the `_id` using Robo 3T or other database tool
1. Checkout this branch
1. Reload the page (the primary contact fields should be gone) and click Get One Resource
1. Copy the `_id` of this new resource
1. Lookup the new resource in the database (in a new tab so you don't lose the first one)
1. Go back to the first resource and grab this line:
`"primaryContactPerson" : ObjectId("5af73cdb2b72c4f89b9a80b5")`
- Note: this is to make sure you can still save the form successfully, even if the resource already has a primary contact
11. Edit the 2nd resource and add the copied line to add a primary contact
12. Go back to the form, change a couple of the fields, and submit
13. Make sure the data saves successfully and without errors
